src/clj/fluree/db/api/transact.cljc

```diff
@@ -70,36 +71,42 @@
 (defn transact-ledger!
   [ledger txn {:keys [expanded? context triples?] :as opts}]
   (go-try
-   (let [txn-context (or (ctx-util/txn-context txn)
-                         context) ;; parent context might come from a Verifiable Credential's context
-         expanded    (if expanded?
-                       txn
-                       (json-ld/expand (ctx-util/use-fluree-context txn)))
-         triples     (if triples?
-                       txn
-                       (q-parse/parse-txn expanded txn-context))
-         parsed-opts (parse-opts expanded opts txn-context)
-         db          (<? (stage-triples (ledger/-db ledger) triples parsed-opts))
-         ;; commit API takes a did-map and parsed context as opts
-         ;; whereas stage API takes a did IRI and unparsed context.
-         ;; Dissoc them until deciding at a later point if they can carry through.
-         cmt-opts    (dissoc parsed-opts :context :did)] ;; possible keys at f.d.ledger.json-ld/enrich-commit-opts
-     (<? (ledger/-commit! ledger db cmt-opts)))))
+    (let [expanded    (if expanded?
```

Comment:
Isn't `transact-ledger!` an internal api? I suspect we can just always pass an expanded txn in without having to bother with this flag.

Reply:
i addressed this in one of my unmerged prs, so i don't think we have to worry about it here